server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+    # GET /ssot/:id
+    get:
+      tags:
+        - RPA-Functionalities
+      summary: Gets all the tasks to execute for an application
+      operationId: getTasks
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Tasks'
+        404:
+          description: Not Found
+  #### Robots ####
+  /robots/{robotId}:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /ssot/:id
+    get:
+      tags:
+        - Robots
+      summary: Get a robot with a specific id
+      operationId: getSpecificRobot
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Robots'
+    # // POST /overwriteRobot/78d09f66d2ed466cf20b06f7
+    put:
+      tags:
+        - Robots
+      summary: Overwrite the existing robot with an updated one
+      operationId: overwriteRobot
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Robots'
+        description: updated Robot object
+        required: true
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+  /robots/{robotId}/robotCode:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /parser/getForId/:robotId
+    get:
+      tags:
+        - Robots
+      summary: Get the robot framework code of a specific robot
+      operationId: getRobotCode
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RobotCode'
+  /robots/{robotId}/robotName:
+    parameters:
+      - name: robotId
+        in: path
+        description: The id of a robot
+        required: true
+        schema:
+          $ref: '#/components/schemas/RobotIds'
+    # GET /renameRobot?id=78d09f66d2ed466cf20b06f7&newName=Bot+Browser
+    patch:
+      tags:
+        - Robots
+      summary: Updates the name of a robot
+      operationId: setRobotName
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RobotNameObjects'
+        description: Object with new robot name
+        required: true
+      responses:
+        204:
+          description: No Content
+        400:
+          description: Bad Request
+  /robots/userAccess:
+    # // GET /shareRobotWithUser?userId=78d09f66d2ed466cf20b06f7&robotId=78d09f66d2ed466cf20b06f7
```

Comment:
same for this comment - its a GET comment, but POST request

Reply:
please see my answer above